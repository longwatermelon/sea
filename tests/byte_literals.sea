fn main() -> int {
    let a:byte = 'A';
    let b:byte = 65b;
    let n:byte = '\n';
    let z:byte = '\0';

    return a + b;
}
