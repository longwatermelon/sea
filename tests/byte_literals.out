RETURN_CODE:130
STDOUT:
